docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -52,25 +52,80 @@ image::images/customize-event-renderer-page.png[Shows the Event Renderer page]
+Each of the tabs within the Alert details flyout are described below.
 
-* *Summary*: Shows an aggregated view of alert details. Alerts that have been enriched with `threat.indicator` data also display the *threat summary* section, which is an additional section located beneath the alert summary. In the *threat summary* section, you can view mapped data for the following `threat.indicator` subfields:
-** `matched.field`
-** `matched.type`
-** `source (threat.indicator.provider)`
-** `first_seen`
-** `last_seen`
+[discrete]
+*Summary tab*
+
+The *Summary* tab provides an overview of the alert and shows relevant threat intelligence details. You can use this information to gain an understanding of what caused the alert and to make decisions about handling it.
+
+The *Summary* tab always begins with a list of general alert details. These details are conveyed as a curated list of fields containing essential and available information about the alert. Within this section, you can find the alert's current status, locate the rule that produced the alert, learn the alert's severity, and more. You can use these details to start forming a sense of what caused the alert and then continue your investigation by checking out available threat intelligence on the alert.
+
+The {es-sec} app collects threat intelligence data for all alerts. Fields that show threat intelligence data are appended with an informational icon and placed after general alert details on the *Summary* tab.
+
+When gathering threat intelligence, the {es-sec} app  queries indicator data from the past 30 days to generate enrichments for the below fields.
+If these field values aren't found, the {es-sec} app does not create threat intelligence for the alert.
+
+NOTE: You can learn more about the query by going to the **Threat Intel** tab and clicking the **Inspect** icon next to threat you want to inspect.
+
+- `file.hash.md5`: The MD5 hash
+- `file.hash.sha1`: The SHA1 hash
+- `file.hash.sha256`: The SHA256 hash
+- `file.pe.imphash`: Imports in a PE file
+- `file.elf.telfhash`: Imports in an ELF file
+- `file.hash.ssdeep`: The SSDEEP hash
+- `source.ip`: The IP address of the source (IPv4 or IPv6)
+- `destination.ip`*: The event's destination IP address
+- `url.full`: The full URL of the event source
+- `registry.path`: The full registry path, including the hive, key, and value
+
+Additional threat intelligence is generated for indicator match alerts. These details are produced when indicators are matched with source events, as outlined by the conditions in the relevant indicator match rule. Any data that's duplicated is removed.
+
+*Threat Intel tab*
+
+Shows the number of threats linked the alert and displays them individually. Threat titles are formed using values from the `matched.field` and `matched.atomic` fields and threats appear in reverse chronological order, with the most recent at the top. Available `threat.indicator` and `source.event` data is displayed for each threat.
```

Comment:
```suggestion
Shows the number of individual threats linked to the alert in reverse chronological order, with the most recent threats at the top. Threat titles are formed using values from the `matched.field` and `matched.atomic` fields. Available `threat.indicator` and `source.event` data is displayed for each threat.
```
Question -- what do you mean by "available `threat.indicator` data is displayed? Is it always there? If so, take out `available`.

Reply:
A few notes here: 

* "linked" is a verb we haven't used previously, I would stick with "matched"
* It might not be clear what a "threat title" is, I would just refer to that heading as a "label"
* I'm skeptical of the utility of that last sentence as that's more of an implementation detail subject to change, and the available fields are self-evident. _Maybe_ you could say "All mapped fields are displayed for each matched threat/indicator." if you want to qualify it.

```suggestion
Shows the number of threats matching the alert in reverse chronological order, with the most recent at the top. Each threat is labelled with values from its `matched.field` and `matched.atomic` fields. All mapped fields are displayed for each matched threat.
```